packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -74,14 +86,49 @@ PageHeaderBreadcrumbBar.displayName = 'PageHeaderBreadcrumbBar';
  * -----------------
  */
 interface PageHeaderContentProps {
+  /**
+   * Provide child elements to be rendered inside PageHeaderContent.
+   */
   children?: React.ReactNode;
+  /**
+   * Specify an optional className to be added to your PageHeaderContent
+   */
   className?: string;
+  /**
+   * Provide an optional icon to render in front of the PageHeaderContent's title.
+   */
+  renderIcon?: ComponentType | FunctionComponent;
+  /**
+   * The PageHeaderContent's title
+   */
+  title: string;
+  /**
+   * The PageHeaderContent's subTitle
+   */
+  subTitle?: string;
```

Comment:
Should the camelCasing be removed since subtitle is one word?

```suggestion
  /**
   * The PageHeaderContent's subtitle
   */
  subtitle?: string;
```